toonz/sources/toonz/exportlevelpopup.cpp

```diff
@@ -387,6 +350,37 @@ ExportLevelPopup::~ExportLevelPopup() {
   QWidget().setLayout(layout());  // Reparents all layout-managed child widgets
 }  // (i.e. all) to a temporary
 
+void ExportLevelPopup::GetSelectedSimpLevels() {
```

Comment:
```suggestion
void ExportLevelPopup::collectSelectedSimpleLevels() {
```

I'd change this from `simp` > `simple` to match `TXshSimpleLevel` wording for readability.